TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/human/HumanActivity.java

```diff
@@ -15,13 +15,11 @@
 public class HumanActivity extends AppCompatActivity {
 
     private static final String LAZERS = String.valueOf(Character.toChars(0x1F4A5));
-    private static final long COMMAND_REPEAT_DELAY = TimeUnit.MILLISECONDS.toMillis(100);
+    private static final long COMMAND_FADING_DELAY = TimeUnit.MILLISECONDS.toMillis(100);
```

Comment:
Maybe it is worth moving this internally to the debug view? Likely will be the same whether ever we use it at this point.